src/flows/pipes/Schedule/ExportTaskButton.tsx

```diff
@@ -33,17 +34,19 @@ const ExportTaskButton: FC<Props> = ({generate}) => {
     })
   }
 
+  const text = `Export as${alert ? ' Alert' : ''} Task`
```

Comment:
this feels wrong

Reply:
totally does. why not just pass in the `text` to the component? it should be more flexible than a boolean in case we decide to add more export task types

Reply:
totally. I was thinking of trying to use the ID to identify whether it should render one string or another. The consideration there is that the title of the overlay should also change to Alert Task rather than Task right? Unless we wanna follow that pattern and have the title of the overlay controlled by a parent passing in a prop?

Reply:
i mean you're injecting a lot of context into this button that it doesn't really need right? you're forcing it to have a concept of an alert, and a task, and whatever else comes down the pipe, when it's not doing anything with those concepts. instead, it'd be cleaner to say "you are a button and you display text" and have the parent push in whatever text needs to show up here.